### PR TITLE
Avoid same minimal filename to get overwritten

### PR DESCRIPTION
**Pull request by x9090**

### CODE REVIEW

**Comment by ivanfratric** [comment]
@0vercl0k since you wrote winafl-cmin, coud you comment on this submission?

**Comment by 0vercl0k** [comment]
Sounds good - I'll take a look at this in the next few days :).

Cheers

2017-11-17 2:53 GMT-08:00 ivanfratric <notifications@github.com>:

File: winafl-cmin.py [hunk 1]
```diff
@@ -292,7 +292,19 @@ def setup_argparse():
     )
     return parser.parse_args()
 
-
+def do_unique_copy(filepath, dest_path):
+    filename = os.path.basename(filepath)
+    new_dest = os.path.join(dest_path, filename)
+    
+    id = 0
```
Comment (0vercl0k) [inline winafl-cmin.py]: Doesn't this override the built-in id() function?

Comment (x9090) [reply winafl-cmin.py]: It does but the id() override only limited within the scope of the function, so i think the concern is pretty trivial.

**Comment by 0vercl0k** [comment]
LGTM!

**Comment by ivanfratric** [comment]
Thanks @x9090 and @0vercl0k !

**Pull request merged**